nncf/experimental/torch/nas/bootstrapNAS/search/search.py

```diff
@@ -459,9 +459,8 @@ def _evaluate(self, x: List[float], out: Dict[str, Any], *args, **kargs) -> NoRe
             self._elasticity_handler.activate_subnet_for_config(sample)
             if sample != self._elasticity_handler.get_active_config():
                 nncf_logger.warning("Requested configuration was invalid")
-                nncf_logger.warning("Requested: {sample}".format(sample=sample))
-                nncf_logger.warning("Provided: {config}".format(
-                                    config=self._elasticity_handler.get_active_config()))
+                nncf_logger.warning(f"Requested: {sample}")
+                nncf_logger.warning(f"Provided: {self._elasticity_handler.get_active_config()}")
```

Comment:
Propose moving to debug
@jpablomch

Reply:
@ljaljushkin Moving to debug works for us. Thanks!

Reply:
Done